src/components/Homepage/subcomponents/FeaturedSection.tsx

```diff
@@ -32,22 +33,26 @@ const FeaturedSection = () => {
         <div className={styles.textContainer}>
           <h2>{FEATURED_TITLE}</h2>
           <p>{FEATURED_DESC}</p>
+          <Link href={`/speakers/${FEATURED_TITLE}`}>
+            <button
```

Comment:
same with here

Reply:
Moved the buttons to a separate client component <RedirectButton /> for all instances.